src/containers/ImagePreviewModalContainer.js

```diff
@@ -0,0 +1,59 @@
+import React, { memo } from 'react'
+import PropTypes from 'prop-types'
+import Modal from 'react-modal'
+
+Modal.setAppElement('#root')
+
+const customStyles = {
```

Comment:
Great! I think one more improvement would be that instead of repeating the entire Modal container with different styles, we could add another set of custom styles in the same container, and then take some boolean attribute as props to decide whether modal should be long or not. There is not much difference with this approach, but I think it's a little bit cleaner to avoid duplicate code. :)

For example:
```

const customStyles = {
  content: {
    top: '50%',
    left: '50%',
    right: 'auto',
    bottom: 'auto',
    marginRight: '-20%',
    transform: 'translate(-50%, -50%)',
    position: 'fixed',
    border: 'none',
    borderRadius: '5px',
    textAlign: 'center',
    maxHeight: '100vh',
  },
  overlay: {
    position: 'fixed',
    zIndex: '2000',
    backgroundColor: 'rgba(0, 0, 0, 0.5)',
  },
}

const customStylesLong = {
  content: {
    top: '50%',
    left: '50%',
    right: 'auto',
    bottom: 'auto',
    marginRight: '-20%',
    transform: 'translate(-50%, -50%)',
    position: 'fixed',
    border: 'none',
    borderRadius: '5px',
    textAlign: 'center',
    maxHeight: '100vh',
  },
  overlay: {
    position: 'fixed',
    zIndex: '2000',
    backgroundColor: 'rgba(0, 0, 0, 0.5)',
  },
}

const ModalContainer = props => {
  const { children, modalIsOpen, closeModal, label, isLong } = props

  return (
    <Modal
      isOpen={modalIsOpen}
      onRequestClose={closeModal}
      contentLabel={label}
      style={isLong ? customStylesLong : customStyles}
      role="dialog"
      aria-labelledby={label}
    >
      {children}
    </Modal>
  )
}
```

Reply:
Changed, thanks!